specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-05-30-preview/share.json

```diff
@@ -785,12 +785,6 @@
           },
           {
             "$ref": "#/parameters/api-version"
-          },
-          {
-            "in": "header",
-            "name": "repeatability-request-id",
-            "description": "If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-Id and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-Id is an opaque string representing a client-generated, globally unique for all time, identifier for the request. It is recommended to use version 4 (random) UUIDs.",
-            "type": "string"
           }
```

Comment:
Any change to an API in public preview, and particularly breaking changes as this one is, must be done in a new api-version.